src/coreComponents/physicsSolvers/contact/SolidMechanicsEmbeddedFractures.cpp

```diff
@@ -725,18 +758,72 @@ void SolidMechanicsEmbeddedFractures::updateState( DomainPartition & domain )
                                             oldJump,
                                             jump,
                                             fractureTraction,
-                                            dFractureTraction_dJump );
+                                            dFractureTraction_dJump,
+                                            fractureState );
       } );
     } );
   } );
 }
 
 bool SolidMechanicsEmbeddedFractures::updateConfiguration( DomainPartition & domain )
 {
-  GEOSX_UNUSED_VAR( domain );
-  return true;
-}
+  bool hasConfigurationConverged = true;
+
+  using namespace extrinsicMeshData::contact;
 
+  forMeshTargets( domain.getMeshBodies(), [&] ( string const &,
+                                                MeshLevel & mesh,
+                                                arrayView1d< string const > const & regionNames )
+  {
+    ElementRegionManager & elemManager = mesh.getElemManager();
+
+    elemManager.forElementSubRegions< EmbeddedSurfaceSubRegion >( regionNames, [&]( localIndex const,
```

Comment:
IIUC this loop checks old and new configuration state to see if it has changed or not?
If there is not change, then you conclude it's a convergence?
Maybe add a little comments about this?

Reply:
yes, precisely.